component/container-runtime.jsonnet

```diff
@@ -6,15 +6,38 @@ local inv = kap.inventory();
 
 local params = inv.parameters.openshift4_nodes;
 
+local mergedConfigs =
+  std.foldl(
+    function(configs, name)
+      local pool = params.machineConfigPools[name];
+      configs {
+        [if std.objectHas(pool, 'containerRuntime') then name]: pool.containerRuntime,
+      },
+    std.objectFields(params.machineConfigPools),
+    {}
+  ) + com.makeMergeable(params.containerRuntimeConfigs);
+
+
 local containerRuntimeConfigs = [
+
   kube._Object('machineconfiguration.openshift.io/v1', 'ContainerRuntimeConfig', nodeGroup) {
+    local fallback(parent, key, obj) = {
+      [if !std.objectHas(parent, key) then key]: obj,
+    },
```

Comment:
This is duplicated verbatim between here, kubelet configs, and machine config pools.